### DISCUSSION TITLE
Takeaways from Micro2Macro 2024

### DISCUSSION THREAD

**Discussion by jtbuch** [open-atmos/PySDM#1409] [Ideas]
Hi @slayoo and @claresinger, I'm starting a discussion here to share a few interesting conversations I had regarding super-droplet microphysics at the [Micro2Macro](https://usclivar.org/meetings/micro2macro) workshop this week:

1. Kamal Kant Chandrakar (NCAR) has included a turbulent collision kernel in the CM1 LES super-droplet code, [reporting](https://www.pnas.org/doi/10.1073/pnas.2319664121) a significant impact on rain initiation in cumulus congestus clouds. Hugh Morrison (NCAR) specifically pointed out that turbulence minimizes the impact of giant CCNs and yields a better agreement with observations than a gravitational kernel.

2. Corey Bois (@CappedColumn), one of Steve Krueger's graduate students, is also looking into incorporating the 1D LEM into PySDM, which models the effect of sub-grid turbulence through stochastic supersaturation fluctuations as I flagged in #1285 A recent [review](https://arxiv.org/abs/2410.03789) benchmarked different turbulence models for cloud-edge mixing using DNS as a reference.

3. There was also research from Robert Wood's (UW) group about giant CCN impacts on precipitation in marine boundary layer clouds where they have an exponentially decaying size distribution truncated at r $\sim$ 10 $\mu {\rm m}$. It reminded me that even in my own experiments with PySDM, collision-coalescence fails for r $\gtrsim$ 10 $\mu {\rm m}$ droplets, since the `GunnKinzer` parameterization for terminal velocity does not extrapolate beyond drizzle drop size of r $\sim 6 {\rm mm}$. An easy fix is to use the `PowerSeries` parameterization instead, but that yields artificially high surface precipitation values.

**Comment by claresinger** [comment]
4. I also discussed with Laura Fierce (PNNL) about surface-active organics. She has a graduate student who is currently working on a new parameterization of surface tension (based on lab experiments) and may be interested in implementing and testing it in PySDM.
